packages/store/src/codegen/tables/TableData.sol

```diff
@@ -0,0 +1,536 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+/* Autogenerated file. Do not edit manually. */
+
+// Import schema type
+import { SchemaType } from "@latticexyz/schema-type/src/solidity/SchemaType.sol";
+
+// Import store internals
+import { IStore } from "../../IStore.sol";
+import { StoreSwitch } from "../../StoreSwitch.sol";
+import { StoreCore } from "../../StoreCore.sol";
+import { Bytes } from "../../Bytes.sol";
+import { Memory } from "../../Memory.sol";
+import { SliceLib } from "../../Slice.sol";
+import { EncodeArray } from "../../tightcoder/EncodeArray.sol";
+import { Schema, SchemaLib } from "../../Schema.sol";
+import { PackedCounter, PackedCounterLib } from "../../PackedCounter.sol";
+
+bytes32 constant _tableId = bytes32(abi.encodePacked(bytes16("mudstore"), bytes16("TableData")));
+bytes32 constant TableDataTableId = _tableId;
+
+struct TableDataData {
```

Comment:
not blocking but "data data" is a little weird

would calling this table `Table` be too weird/confusing?

Reply:
we'd get `TableTableId`

Reply:
what about `TableMetadata` or `StoreMetadata`? (since the old table can go away, and the new one can get an extra field - `tableName`)